src/Compute/Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetCreateOrUpdateMethod.cs

```diff
@@ -176,14 +176,14 @@ public override void ExecuteCmdlet()
                                 var res = VirtualMachineScaleSetsClient.CreateOrUpdateWithHttpMessagesAsync(
                                         resourceGroupName,
                                         vmScaleSetName,
-                                        parameters,null,null,
+                                        parameters, this.IfMatch, this.IfNoneMatch,
```

Comment:
Make sure to run tests where these are null to ensure this does not throw a null ref exception.

Reply:
here is running with ifMatch and ifNoneMatch not provided: 
![image](https://github.com/Azure/azure-powershell/assets/10857691/798af67a-501c-44ae-8301-ff2e9e2a586c)
![image](https://github.com/Azure/azure-powershell/assets/10857691/513166bc-93a6-40f0-b89a-839192c69de3)
![image](https://github.com/Azure/azure-powershell/assets/10857691/94e34c63-40d0-4acc-976b-0dafb37b0434)